app/build.gradle

```diff
@@ -188,4 +189,5 @@ dependencies {
 }
 
 apply plugin: 'ManifestClasspath'
-apply plugin: 'com.google.gms.google-services'
\ No newline at end of file
+apply plugin: 'com.google.gms.google-services'
+apply plugin: 'kotlin-android-extensions'
```

Comment:
I wouldn't like to include this since i'm having Butterknife and looking at your PR you are mixing both having onClick from butterknife but including the layout ids statistically. 

Follow any Activity that is written in kotlin